### ISSUE TITLE
`plot_digits.py` => `no attribute 'index'`

### ISSUE THREAD

**Issue by hughperkins** [ContextLab/hypertools#85]
```
(env3) (master|✔) ~/git/hypertools/examples$ python plot_digits.py 
Traceback (most recent call last):
  File "plot_digits.py", line 21, in <module>
    hyp.plot(data, 'o', group=group)
  File "/home/ubuntu/env3/lib/python3.5/site-packages/hypertools/plot/plot.py", line 191, in plot
    x = reshape_data(x,group)
  File "/home/ubuntu/env3/lib/python3.5/site-packages/hypertools/_shared/helpers.py", line 127, in reshape_data
    categories = list(sorted(set(labels), key=labels.index))
AttributeError: 'numpy.ndarray' object has no attribute 'index'
```
My packages:
```
(env3) (master|✔) ~/git/hypertools/examples$ python -m pip freeze
appdirs==1.4.0
cycler==0.10.0
future==0.16.0
hypertools==0.1.5
matplotlib==1.5.3
numpy==1.12.0
packaging==16.8
pandas==0.19.2
pkg-resources==0.0.0
ppca==0.0.2
pyparsing==2.1.10
python-dateutil==2.6.0
pytz==2016.10
scikit-learn==0.18.1
scipy==0.18.1
seaborn==0.7.1
six==1.10.0
```

python version:
```
(env3) (master|✔) ~/git/hypertools/examples$ python -V
Python 3.5.2
```

**Comment by andrewheusser** [comment]
should be fixed now 4930a7a, thanks :)

**Comment by hughperkins** [comment]
Thanks! :-)  Will take a look.
